doc/tagbar.txt

```diff
@@ -1101,6 +1101,9 @@ TagbarScope
 TagbarType
     The type of a tag or scope if available.
 
+TagbarTagLineN
+    The line num of a tag if set it on at right.
```

Comment:
This needs some English copy editing.

```suggestion
    The source line number displayed to the right of each tag entry.
```

Reply:
um... ok, i am not native english speaker, if you felt that's better, then ok.